src/ansible/playbook_vm.yml

```diff
@@ -8,11 +8,13 @@
 - name: Include base
   ansible.builtin.import_playbook: playbook_image_base.yml
   vars:
+    passkey_support: no
```

Comment:
How are you going to test passkey on RHEL?

Reply:
Well as we discussed before, we do not want to pull the whole build environment that is needed for passkey by default. There was discussion about packaging the paskey pieces in copr an pulling them when needed instead.